tests/cucumber/steps/steps.js

```diff
@@ -5341,6 +5341,58 @@ module.exports = function getSteps(options) {
     }
   );
 
+  Given('a source map json file {string}', async function (srcmap) {
+    const js = parseJSON(await loadResource(srcmap));
+    this.sourcemap = new algosdk.SourceMap(js);
+  });
+
+  Then(
+    'the string composed of pc:line number equals {string}',
+    function (mapping) {
+      const buff = [];
+      for (const [pc, line] of Object.entries(this.sourcemap.pcToLine)) {
+        buff.push(`${pc}:${line}`);
+      }
```

Comment:
@barnjamin Optional:  The following statements _can_ be written more concisely using higher order functions.

I don't feel strongly.  Primarily, I tested the refactoring locally to further my understanding of TypeScript language features.  If you prefer as is, feel welcomed to resolve.

```suggestion
      const buff = Object.entries(this.sourcemap.pcToLine).map(([pc, line]) => `${pc}:${line}`);
```